src/view/actions/sendEvent.jsx

```diff
@@ -113,101 +114,123 @@ const knownEventTypeOptions = [
-                    width="size-5000"
-                  >
-                    {item => <Item key={item.type}>{item.type}</Item>}
-                  </ComboBox>
-                </DataElementSelector>
-                <DataElementSelector>
-                  <TextField
-                    data-test-id="xdmField"
-                    name="xdm"
-                    label="XDM Data"
-                    description="Provide a data element which returns an object matching your XDM schema."
-                    width="size-5000"
-                  />
-                </DataElementSelector>
-                <DataElementSelector>
-                  <TextField
-                    data-test-id="dataField"
-                    name="data"
-                    label="Data"
-                    description="Provide a data element which returns an object to send as free-form data."
-                    width="size-5000"
-                  />
-                </DataElementSelector>
-                <DataElementSelector>
-                  <TextField
-                    data-test-id="mergeIdField"
-                    name="mergeId"
-                    description="Provide an identifier used to merge multiple events. This will
-                      populate the `eventMergeId` XDM field."
-                    label="Merge ID"
-                    width="size-5000"
-                  />
-                </DataElementSelector>
-                <DataElementSelector>
-                  <TextField
-                    data-test-id="datasetIdField"
-                    name="datasetId"
-                    description="Send data to a different dataset than what's been provided in the Edge configuration."
-                    label="Dataset ID"
-                    width="size-5000"
-                  />
-                </DataElementSelector>
-                <Checkbox
-                  data-test-id="documentUnloadingField"
-                  name="documentUnloading"
-                  description="Check this to ensure the event will reach the server even if the user is navigating away from the current document (page). Any response from the server will be ignored."
-                  width="size-5000"
-                >
-                  Document will unload
-                </Checkbox>
-                <Checkbox
-                  data-test-id="renderDecisionsField"
-                  name="renderDecisions"
-                  description="Check this to automatically render personalization and pre-hide the content to prevent flicker."
-                  width="size-5000"
-                >
-                  Render visual personalization decisions
-                </Checkbox>
-                <DecisionScopes />
-              </Form>
-            )}
+            render={props => {
+              return <SendEvent {...props} />;
+            }}
           />
         );
       }}
     />
   );
 };
 
-render(SendEvent);
+render(SendEventExtensionView);
```

Comment:
The changes to SendEvent were just to accommodate the refactored ExtensionViewForm.